web/index.html

```diff
@@ -676,21 +819,26 @@ <h3>Used in</h3>
                 });
 
                 document.addEventListener("wheel", (event) => {
+                    this.isScrolling = true;
                     window.lastScrollTime ??= Date.now();
                     if (this.currentElement && window.lastScrollTime + 200 < Date.now()) {
-                        if (event.deltaX > 20) {
+                        if (event.deltaX > 30 && event.deltaY < 30) {
                             this.slideRight()
                         }
-                        if (event.deltaX < -20) {
+                        if (event.deltaX < -30 && event.deltaY < 30) {
                             this.slideLeft()
```

Comment:
updated the sideways scrolling amounts need to switch panes, and made sure that the user was scrolling more sideways than down for that to happen, because I was having the panes switch non-stop for me whenever I was scrolling on my laptop without wanting them to and it was really annoying.